slides/index.html

```diff
@@ -258,6 +258,49 @@
 ---
 class: center, middle
 
+# LF oppgave 2
+
+```cpp
+puck->addCharacteristic(
+  SAMPLE_GATT_SERVICE,
+  SAMPLE_GATT_CHARACTERISTIC, 20,
+  GattCharacteristic::
```

Comment:
This parameter defaults to this value, so it can be omitted if you want. I'm not sure what's best in terms of teaching.